tests/test_viewing.py

```diff
@@ -329,9 +330,9 @@ def test_view_supported_index_types(module):
             assert module.comp(index), f"Failed for {type(index)}"
             assert View(module).comp(index), f"Failed for {type(index)}"
 
-            # for loc test float and list of floats
-            assert module.loc(0.0), "Failed for float"
-            assert module.loc([0.0, 0.5, 1.0]), "Failed for List[float]"
+        # for loc test float and list of floats
+        assert module.loc(0.0), "Failed for float"
+        assert module.loc([0.0, 0.5, 1.0]), "Failed for List[float]"
     else:
         with pytest.raises(AssertionError):
             module.comp(0)
```

Comment:
Could we have three more tests:

1) for correctness of the boolean indexing.
2) For allowing boolean indexing of synapses.
3) Combining select with boolean indexing.